### PR TITLE
chore: Remove ignored for fixed vulnerabilities

### PR DESCRIPTION
**Pull request by markstory**
We no longer need to ignore these problems as we're on a newer version of Django.

Refs SEN-621

### CODE REVIEW

**Review by mattrobenolt — APPROVED** [review]
We did it. 💯

File: bin/scan [hunk 1]
```diff
@@ -5,30 +5,6 @@ ignored=(
...
-    # Reason: We don't leverage this feature.
-    25714
-    # django, installed 1.6.11, affected <1.7.6, id 25715
-    # Description: Cross-site scripting (XSS) vulnerability in the contents function in admin/helpers.py in Django before 1.7.6 and 1.8 before 1.8b2 allows remote attackers to inject arbitrary web script or HTML via a model attribute in ModelAdmin.readonly_fields, as demonstrated by a @property.
-    # Reason: We don't leverage this feature.
-    25715
-    # django, installed 1.6.11, affected <1.8.10, id 33073
-    # Description: The utils.http.is_safe_url function in Django before 1.8.10 and 1.9.x before 1.9.3 allows remote attackers to redirect users to arbitrary web sites and conduct phishing attacks or possibly conduct cross-site scripting (XSS) attacks via a URL containing basic authentication, as demonstrated by http://mysite.example.com\@attacker.com.
-    # Reason: We have already patched around this and use our own `is_safe_url` function pulled from Django 1.10.
```
Comment (EvanPurkhiser) [inline bin/scan]: I assume we removed the monkey patch already?

Comment (mattrobenolt) [reply bin/scan]: Oh, no, I doubt it. But it doesn't particularly for this context. Probably just good hygiene anyways.

Comment (markstory) [reply bin/scan]: No. I'll track that down and see if it can be excised.

Comment (markstory) [reply bin/scan]: I took a look and the `is_safe_url` we're using comes from Matt's [django-sudo](https://github.com/mattrobenolt/django-sudo/blob/master/sudo/utils.py) which we're using for a few other things. There isn't anything to remove from sentry at this point in time.

**Pull request merged**